app/docker/views/networks/types.ts

```diff
@@ -0,0 +1,31 @@
+export type NetworkId = string;
+
+export interface DockerNetwork {
+  Name: string;
+  Id: string;
+  Driver: string;
+  Scope: string;
+  Attachable: boolean;
+  Internal: boolean;
+}
+
+export type NetworkKey =
+  | 'Name'
+  | 'Id'
+  | 'Scope'
+  | 'Driver'
+  | 'Attachable'
+  | 'Internal';
+
+type NetworkValue = string;
+
+export type NetworkRowContent = [NetworkKey, NetworkValue][];
+
+type IPConfig = {
+  Subnet: string;
+  Gateway: string;
+  IPRange: string;
+  AuxiliaryAddresses: string[];
+};
+
+export type IPConfigs = IPConfig[];
```

Comment:
alias for array is a bit redundant. just use `IPConfig[]` wherever you need this

Reply:
updated